src/.vuepress/components/ExtensionsList.vue

```diff
@@ -31,23 +32,30 @@ export default {
 
 	data() {
 		return {
+			// List of available sources
 			extensions: [],
-			last_id: "",			// id of the last source. Used to removed the last coma
 		};
 	},
 
 	async beforeMount() {
+		// Get the versioning.json file of the repo
 		const { data } = await axios.get(this.$props.url + "/versioning.json");
 		this.$data.extensions = data["sources"];
-		this.$data.last_id = data["sources"][data["sources"].length - 1];
 	},
-methods: {
-		iconUrl(extensionId, fileName) {
-			//https://paperback-ios.github.io/extensions/${extension_id}/includes/${file_name}
-			return this.$props.url + "/" + extensionId + "/includes/" + fileName;
+	methods: {
+		getName(sourceName) {
+			// Return sourceName without the intext badges (18+, Country-Proof...)
+			return sourceName.replace(" (18+)", "").replace(" (Country-Proof)", "").replace("[FR] ", "")
```

Comment:
After these extension tags are added, we shouldn't display intext badges at all through raw text like this.
Why don't we blanket remove anything in `( )` and `[ ]`, and enforce that if the user want tags, they should add it through the tag system?

```suggestion
			return sourceName.replace(/[(|[][\w|\d\ ]+[)|\]]/g, "").trim()
```

Reply:
Interesting idea. In my opinion this will improve greatly the component.

However I believe their is an issue with special characters like `+` or `-`.

Could you add these characters to the regex? `\[(|[][\w|\d|\ |+|-]+[)|\]]\g` could work for `(18+)` and `(Country-Proof)` but that's probably not the best nor cleanest way to achieve the selection.